.gitignore

```diff
@@ -1,3 +1,11 @@
+# ignore this from code 
+env
+*.pyc
+__pycache__
+.vscode
+.idea
+
+
```

Comment:
why was this added? it should be in the rest of the git ignore file... as this git ignore file was made by GitHub.